exchange/bitswap/bitswap.go

```diff
@@ -417,3 +417,7 @@ func (bs *Bitswap) GetWantlist() []key.Key {
 	}
 	return out
 }
+
+func (bs *Bitswap) IsOnline() bool {
```

Comment:
It doesnt look like we use this anywhere. Am i missing something or is this just dead code from a different approach?